modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/data_renderers/index.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+// @ts-ignore
+
+import ActionsLinkRenderer from './ActionLinkRenderer';
+
+// @ts-ignore
+
+import BooleanRenderer from './BooleanRenderer';
+
+// @ts-ignore
+
+import DateRenderer from './DateRenderer';
+
+// @ts-ignore
+
+import DateTimeRenderer from './DateTimeRenderer';
+
+// @ts-ignore
+
+import DefaultRenderer from './DefaultRenderer';
+
+// @ts-ignore
+
+import ImageRenderer from './ImageRenderer';
+
+// @ts-ignore
+
+import LabelRenderer from './LabelRenderer';
+
+// @ts-ignore
+
+import LinkRenderer from './LinkRenderer';
+
+// @ts-ignore
+
+import QuantitySelectorRenderer from './QuantitySelectorRenderer';
+
+// @ts-ignore
```

Comment:
Whoa, lots of ignores. Is there a ticket out here for us to update those files to TS? some look simple enough, but they would be nice to have since they define certain props